string mode
float32 v_forward
float32 w_yaw
---
string outcome